Automattic/wc-calypso-bridge
# Track calypso_wooexpress_one_dollar_offer when a user clicks on My Ho…

…me and has an active intro offer

### Changes proposed in this Pull Request:



Closes #1322  .






### How to test the changes in this Pull Request:




1. Sign up for a new trial site.
2. Make your trial site SSH-able by following peapX7-1D4-p2
3. Clone this branch and run `npm run start` to build the changes.
4. SSH into your trial site and copy over `/build` directory to `~/htdocs/wp-content/mu-plugins/wpcomsh/vendor/automattic/wc-calypso-bridge/build`
5. Access the site and open browser inspector then paste `window.localStorage.setItem( 'debug', 'calypso:analytics' )`
6. Change inspector log level to verbose
7. Go to WooCommerce Home.
8. Confirm `calypso_wooexpress_one_dollar_offer` track has been fired.

Note: This won't work if a user clicks on `My Home` from a page where the `wcCalypsoBridge` JS variable is not available, such as on the My Mailboxes or Upgrades pages.



### Other information:




### FOR PR REVIEWER ONLY:
